src/Raven.Server/Documents/Includes/Sharding/IncludeDocumentsOrchestratedSubscriptionCommand.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Includes.Sharding;
+
+public class IncludeDocumentsOrchestratedSubscriptionCommand : IncludeDocumentsCommand
+{
+    private readonly ClusterOperationContext _clusterContext;
+    private readonly CancellationToken _token;
+    private List<BlittableJsonReaderObject> _includes;
+
+    public IncludeDocumentsOrchestratedSubscriptionCommand(ClusterOperationContext clusterContext, CancellationToken token)
```

Comment:
would `JsonOperationContext` will be enough here?

Reply:
done